modules/k8s/event.go

```diff
@@ -0,0 +1,34 @@
+package k8s
+
+import (
+	"context"
+
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/gruntwork-io/terratest/modules/testing"
+)
+
+// ListEvents will retrieve the Events in the given namespace that match the given filters and return them. This will fail the
+// test if there is an error.
+func ListEvents(t testing.TestingT, options *KubectlOptions, filters metav1.ListOptions) []corev1.Event {
```

Comment:
Will be helpful to have tests for `ListEvents` to track that in the future function will continue to work

Reply:
Thank you for your review @denis256 . I've done my tests with kubectl and it turns out that the events API doesn't return an error even if you provide the wrong namespace/filters. 

That's why I rewrote the current tests to use the `ListEvents` function and also added a new test which makes sure that `ListEventsE` returns a `nil` error.  What do you think?